core/src/main/java/org/kiwix/kiwixmobile/core/bookmark/BookmarksActivity.java

```diff
@@ -84,6 +84,7 @@
     @Override
     public boolean onCreateActionMode(ActionMode mode, Menu menu) {
       mode.getMenuInflater().inflate(R.menu.menu_context_delete, menu);
+      bookmarksSwitch.setVisibility(View.INVISIBLE);
```

Comment:
Instead of making it invisible, maybe we could just disable the buttons in both history and bookmarks? If we make it invisible, a small black bar will still be visible below the actionMode which looks a bit peculiar.

Reply:
Yeah you are right. Changed it.

Reply:
Cool :+1: